dd-java-agent/instrumentation/javax-mail/src/main/java/datadog/trace/instrumentation/javax/mail/JavaxMailBodyInstrumentation.java

```diff
@@ -0,0 +1,66 @@
+package datadog.trace.instrumentation.javax.mail;
+
+import static datadog.trace.agent.tooling.bytebuddy.matcher.HierarchyMatchers.implementsInterface;
+import static datadog.trace.agent.tooling.bytebuddy.matcher.NameMatchers.named;
+
+import com.google.auto.service.AutoService;
+import datadog.trace.agent.tooling.Instrumenter;
+import datadog.trace.agent.tooling.InstrumenterModule;
+import datadog.trace.api.iast.InstrumentationBridge;
+import datadog.trace.api.iast.Propagation;
+import datadog.trace.api.iast.propagation.PropagationModule;
+import javax.mail.Part;
+import net.bytebuddy.asm.Advice;
+import net.bytebuddy.description.type.TypeDescription;
+import net.bytebuddy.matcher.ElementMatcher;
+
+@AutoService(InstrumenterModule.class)
+public class JavaxMailBodyInstrumentation extends InstrumenterModule.Iast
+    implements Instrumenter.ForTypeHierarchy, Instrumenter.HasMethodAdvice {
+
+  public JavaxMailBodyInstrumentation() {
+    super("javax-mail", "body");
```

Comment:
```suggestion
    super("javax-mail", "javax-mail-body");
```

More  specific alias, since `body` might overlap with different concepts in other instrumentations.